airflow/www/api/experimental/endpoints.py

```diff
@@ -184,13 +184,11 @@ def task_info(dag_id, task_id):
     return jsonify(fields)
 
 
-# ToDo: Shouldn't this be a PUT method?
-@api_experimental.route('/dags/<string:dag_id>/paused/<string:paused>', methods=['GET'])
+@api_experimental.route('/dags/<string:dag_id>/pause/<string:pause_state>', methods=['PUT'])
```

Comment:
POST is more typical here -- it's not wrong here, but I don't think we use PUT anywhere else in the current experimental API do we?

Can you also add a note to UPDATING.md, and then we can pull this in to 1.10.10